src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -91,6 +94,14 @@ ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
                    [this]() { emit InstanceReloadRequested(); });
 
   SetupStateMachine();
+
+  if (absl::GetFlag(FLAGS_enable_project_selection)) {
+    ui_->instancesFilterLineEdit->hide();
+    ui_->refreshButton->hide();
+  } else {
+    // hide() does not work on QLayouts, hence its removed from the parent
```

Comment:
I'd recommend replacing the `QLayout` with a generic `QWidget` to achieve this - it's a nitpick here since this is only required for the feature flag, but generally if you plan on modifying the UI in code, it's better to use a widget instead of meddling with the layouts.